.gitignore

```diff
@@ -16,3 +16,6 @@ generated.rs
 /build
 /target
 /pconvert_rust.egg-info
+
+node_modules/

```

Comment:
Do it similarly to https://github.com/ripe-tech/ripe-util-vue/blob/master/.gitignore (yarn lock, order, etc)

Reply:
I cant find the criteria by looking into our gitignores :/

Reply:
That makes 2 of us lol. Just add the yarn lock then